editor.planx.uk/src/@planx/components/List/Public/Fields.tsx

```diff
@@ -49,7 +49,7 @@ export const TextFieldInput: React.FC<Props<TextField>> = ({
         rows={
           data.type && ["long", "extraLong"].includes(data.type) ? 5 : undefined
         }
-        name={`userData[${activeIndex}][${data.fn}]`}
+        name={`userData[${activeIndex}]['${data.fn}']`}
```

Comment:
This syntax is a bit weird / complex, but it's what Formik requires! Docs: https://formik.org/docs/guides/arrays#avoid-nesting